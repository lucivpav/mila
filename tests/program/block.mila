var a,b : integer;
begin
  a := 1; b := 2;
  if a = b then
  begin
    writeln(a);
    writeln(b);
  end;
end.
---input---
var a,b : integer;
begin
  a := 1; b := 2;
  if a = b then
  begin
    writeln(a);
    writeln(b);
  end
end.
---input---
var a,b : integer;
begin
  a := 1; b := 2;
  if a = b then
  begin
    writeln(a);
    writeln(b);
  end.
end.
---input---
var a,b : integer;
begin
  a := 1; b := 2;
  if a <> b then
  begin
    writeln(a);
    if a = b then
    begin
      writeln(666);
      writeln(666);
    end;
    writeln(b);
  end;
end.
