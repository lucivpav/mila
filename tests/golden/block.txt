0
---output---
0
---output---
256
---output---
1
2
0
